Sources/Rendering/Core/ScalarBarActor/index.d.ts

```diff
@@ -27,21 +27,10 @@ export interface IStyle {
  *
  */
 export interface IScalarBarActorInitialValues extends IActorInitialValues {
-	automated?: boolean,
-	autoLayout?: (publicAPI: object, model: object) => void,
-	axisLabel?: string,
-	barPosition?: Vector2,
-	barSize?: Size,
-	boxPosition?: Vector2,
-	boxSize?: Size,
-	scalarToColors?: null,
-	axisTitlePixelOffset?: number,
-	axisTextStyle?: IStyle,
-	tickLabelPixelOffset?: number,
-	tickTextStyle?: IStyle,
-	drawNanAnnotation?: boolean,
-	drawBelowRangeSwatch?: boolean,
-	drawAboveRangeSwatch?: boolean,
```

Comment:
whydo you get rid of those values ?

Reply:
Total mistake, not sure how that happened. Thanks for catching that.